xds/src/main/java/io/grpc/xds/ClientLoadCounter.java

```diff
@@ -54,27 +54,10 @@
   private final AtomicLong callsIssued = new AtomicLong();
   private final MetricRecorder[] metricRecorders = new MetricRecorder[THREAD_BALANCING_FACTOR];
 
-  // True if this counter continues to record stats after next snapshot. Otherwise, it will be
-  // discarded.
-  private boolean active;
```

Comment:
As a reminder, `active` is true if this counter is currently in-use by some LB policy to track loads sent to a specific locality.

We do not mark internally if a counter is still in use by some LB policy, instead ref-count the usage of this counter externally by wrapping a counter with `ReferenceCounted`.